src/main/java/com/opencart/steps/MainPageBL.java

```diff
@@ -1,12 +1,34 @@
 package com.opencart.steps;
 
+import com.opencart.navigation.Navigation;
+
+import static com.opencart.enums.URLs.BASE_URL;
+
 public class MainPageBL {
 
     public HeaderPageBL getHeaderPageBL() {
         return new HeaderPageBL();
     }
 
-    public void verifyLogout() {
+    public void registerNewUser() {
+        new Navigation().navigateToUrl(BASE_URL.getValue());
+        getHeaderPageBL()
+                .clickOnMyAccountButton()
+                .clickOnRegisterButton()
+                .registerNewPerson()
+                .verifyUserRegistration();
+        logout();
+    }
+
+    public void logout() {
+        getHeaderPageBL()
+                .clickOnMyAccountButton()
+                .clickOnLogoutButton()
+                .clickContinue();
+        verifyLogout();
+    }
```

Comment:
You need to perform all necessary business actions from the test (before/after are no exception)